packages/integration-tests/graphql-codegen-joist.js

```diff
@@ -0,0 +1,14 @@
+const mappers = {
```

Comment:
The integration-tests don't actually use any of these new files, i.e. it hasn't turned into a graphql project, I'm just using it to watch for the output.

Eventually we'll need to have like N `integration-test/foo` directories where each `foo` is some variation of a project config / schema design / etc., and we could have one of them that was very specifically an apollo graphql project or what not.